code/modules/antagonists/blob/powers.dm

```diff
@@ -365,6 +365,7 @@
 		strain_choices = list()
 
 		var/list/new_strains = GLOB.valid_blobstrains.Copy()
+		LAZYREMOVE(new_strains, blobstrain.type)
```

Comment:
No need for this, just change the above line to `var/list/new_strains = GLOB.valid_blobstrains.Copy() - blobstrain.type`.

Reply:
Done